testnet/stacks-node/src/tests/mod.rs

```diff
@@ -109,11 +110,15 @@ pub fn gen_random_port() -> u16 {
     let mut rng = rand::thread_rng();
     let range_len = (1024..u16::MAX).len();
     loop {
+        // Note it needs to be +1 because we reserve one port for the event observer
         assert!(
-            USED_PORTS.lock().unwrap().len() < range_len,
+            USED_PORTS.lock().unwrap().len() + 1 < range_len,
             "No more available ports"
         );
         let port = rng.gen_range(1024..u16::MAX); // use a non-privileged port between 1024 and 65534
+        if port == EVENT_OBSERVER_PORT {
```

Comment:
Should we just initialize `USED_PORTS` to include the `EVENT_OBSERVER_PORT` instead?

Reply:
Yes. :) Done!